spec/features/reservations_spec.rb

```diff
@@ -545,4 +545,24 @@
       end
     end
   end
+  context 'redirects properly' do
+    before(:each) { empty_cart }
+    it 'handles direct url visit to /reservation/new with empty cart' do
+      visit root_path
```

Comment:
Why don't we start on a different page (e.g. `visit equipment_model_path(@eq_model)`)? We could actually start both tests from that page since the equipment model show view has the reserve button on it and therefore add that `visit` call to the `before` block instead.